docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+This demo will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH denominated in USD. For example, if the user wants to withdraw $10,000 USD of wETH, and the ETH/USD exchange rate is $2,000, they would withdraw 5 wETH.
+
+* The user links the `OptimisticDepositBox` with one of the price identifiers enabled on the DVM.
+
+* The user deposits wETH into the `OptimisticDepositBox` and register it with the `ETH/USD` price identifier.
+
+* The user can now withdraw a USD-denominated amount of wETH from their `DepositBox` via smart contract calls, with the Optimistic Oracle enabling optimistic on-chain pricing.
+
+In this example, the user would not have been able to transfer USD-denominated amounts of wETH without referencing an off-chain `ETH/USD` price feed. The Optimistic Oracle therefore enables the user to "pull" a reference price.
+
+Unlike price requests to the DVM, a price request to the Optimistic Oracle can be resolved within a specified liveness window if there are no disputes, which can be significantly shorter than the DVM voting period. The price requestor is not currently required to pay fees to the DVM.
```

Comment:
Two things:
- Putting some numbers around the OO vs DVM speed. Typical 2 hour settlement vs 2-3 day.
- Should we recommend that "most" people pay fees on requests? Looks like the fee payer logic is not in the tutorial, so maybe not.

For future iterations of this tut, I think it'd be good for us to explain why different liveness/rewards amounts should be set, and possibly show how to do that as well.

Reply:
I think we're moving away from the `FeePayer` logic towards collecting fees from contracts with large TVL as needed. Agreed on future interations. This will also need to be updated once we start collecting fees from contracts calling the OO and figure out how that usually works.

Reply:
Makes sense for the fee payer. Fine leaving it out as it simplifies things.